<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>614</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>614</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LAB3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="hashTableWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Предмет</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>161</width>
      <height>396</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="itemLabel">
       <property name="text">
        <string>Предмет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="itemLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Цена:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="priceLineEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить/Изменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delAllButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Удалить всё</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sumValueLabel">
       <property name="text">
        <string>Суммарная стоимость:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sumValueLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>Имя файла:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameLineEdit">
       <property name="text">
        <string>hashtable.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="writeToFileButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Записать в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="readFromFileButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Прочитать из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Записано</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
